model-optimizer/setup.py

```diff
@@ -48,23 +52,21 @@ def run(self):
         install.run(self)
         # Create requirements.txt files for all the frameworks
         for name in requirements_txt:
-            path = os.path.join(self.install_purelib, PACKAGE_NAME, name)
+            path = os.path.join(self.install_purelib, prefix, name)
             with open(path, 'wt') as common_reqs_file:
                 common_reqs_file.write('\n'.join(deps))
         # Add version.txt if exists
         version_txt = 'version.txt'
         if os.path.exists(version_txt):
             copyfile(os.path.join(version_txt),
-                     os.path.join(self.install_purelib,
-                                  PACKAGE_NAME, version_txt))
+                     os.path.join(self.install_purelib, prefix, version_txt))
 
-        path = os.path.join(self.install_purelib, PACKAGE_NAME, '__init__.py')
+        path = os.path.join(self.install_purelib, prefix, '__init__.py')
         with open(path, 'wt') as init_file:
             init_file.write('import os, sys\n')
-            init_file.write('from {} import mo\n'.format(PACKAGE_NAME))
+            init_file.write('from openvino.tools.mo import mo \n')
```

Comment:
Is it done for backward compatibility? I believe we do not need this workaround with remapping modules in __init__.py created from setup.py script. Instead of that python modules should be correctly allocated inside  /openvino/tools/mo directory structure. @dkurt, please to comment.